hugo/layouts/index.html

```diff
@@ -105,14 +105,14 @@ <h3 class="card-title" id="common-name">{{ .Params.subtitle | markdownify }}</h3
     const speciesCards = Array.from(document.querySelectorAll('.scilife-species-card'));
 
     function sortAlphabetically(a, b) {
```

Comment:
A nitpick: it could be worth handling the case where `.querySelector` returns `null`. This would currently raise an attribute error. A possible way to go (although certainly not definitive) : define a `getElementText(root, selector)` function that returns the text content of the element under root retrieved by given selector, or the empty string if no such element exists. Or just using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) maybe,

Reply:
I think the console message is fine. I would perhaps use `console.warn`, and mention `root` as well in some way or another to help pinpoint the faulty DOM node.

My ideal would be an array of JSON objects that acts as a source of truth, which is rendered every time a sorting is requested. But this is way over-engineered for our purposes.

Reply:
Thanks for the suggestions, I've just done that now in commit f459afa25720274945bceac9172cebe5afd7c220